client/src/prelude/tc.res

```diff
@@ -24,6 +26,9 @@ include (
     and module Map := Tablecloth.Map
 )
 
+// I'm not a big fan of these quote-style functions
```

Comment:
In Bucklescript, when using the Ocaml syntax, these were regular infix functions. Now that they are much more difficult to use, no point in having them